pkgs/development/python-modules/python3-msrplib/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, isPy3k
+, buildPythonPackage
+, fetchFromGitHub
+, python3-application
+, python3-eventlib
+, python3-gnutls
+, twisted
+}:
+
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.20.1";
+
+  disabled = !isPy3k;
```

I suggest to limit the supported Python releases to the ones which are not EOL.

```suggestion
  disabled = pythonOlder "3.7";
```